x-pack/legacy/plugins/canvas/i18n/components.ts

```diff
@@ -121,4 +125,77 @@ export const ComponentStrings = {
         defaultMessage: 'Show editing controls',
       }),
   },
+  WorkpadHeaderAutoRefreshControls: {
+    getRefreshListTitleText: () =>
+      i18n.translate('xpack.canvas.workpadHeaderAutoRefreshControls.refreshListTitleText', {
+        defaultMessage: 'Refresh elements',
+      }),
+    getRefreshListDurationPrefix: () =>
+      i18n.translate('xpack.canvas.workpadHeaderAutoRefreshControls.refreshListDurationPrefix', {
+        defaultMessage: 'Every',
+      }),
+    getRefreshListDurationManualText: () =>
+      i18n.translate(
+        'xpack.canvas.workpadHeaderAutoRefreshControls.refreshListDurationManualText',
+        {
+          defaultMessage: 'Manually',
+        }
+      ),
+    getDisableTooltipText: () =>
+      i18n.translate('xpack.canvas.workpadHeaderAutoRefreshControls.disableTooltipText', {
+        defaultMessage: 'Disable auto-refresh',
+      }),
+    getIntervalFormLabelText: () =>
+      i18n.translate('xpack.canvas.workpadHeaderAutoRefreshControls.intervalFormLabel', {
+        defaultMessage: 'Change auto-refresh interval',
+      }),
+  },
+  WorkpadHeaderControlSettings: {
+    getTooltipText: () =>
+      i18n.translate('xpack.canvas.workpadHeaderControlSettings.tooltipText', {
+        defaultMessage: 'Control settings',
+      }),
+  },
+  WorkpadHeaderCustomInterval: {
+    getFormLabelText: () =>
+      i18n.translate('xpack.canvas.workpadHeaderCustomInterval.formLabelText', {
+        defaultMessage: 'Set a custom interval',
+      }),
+    getFormHelpText: () =>
+      i18n.translate('xpack.canvas.workpadHeaderCustomInterval.formHelpText', {
+        defaultMessage: 'Use shorthand notation, like 30s, 10m, or 1h',
```

Comment:
Not sure how we want to handle text like this. The time interval function that will receive this form input uses `s`, `m`, and `h` for its time duration calculations but that doesn't necessarily seem "i18n friendly". If we require `s`, `m`, and `h` as units, should the examples given here be non-translated constants?